### PR TITLE
fix(UserListPanel, ContactsColumnView): Proper bottom margin when the list is not positioned at the end

### LINKED ISSUE #7526: Community members and chat lists should end on the bottom edge when not positioned at the end
# Bug Report

## Description

When the list is positioned at the end, the margin is proper. However, when the list is not at the end, it's truncated too soon.

Image

## Steps to reproduce

Go to community with high number of members.

#### Expected behavior

When the list is not at the end, it should end at the bottom line.

#### Actual behavior

The list is truncated too soon.

### Additional Information

- Status desktop version:
- Operating System:

### PR DESCRIPTION
**Pull request by micieslak**
### What does the PR do

Bottom margin of the lists is fixed (when the requires scrolling and is not positioned at the end).

Closes: https://github.com/status-im/status-desktop/issues/7526

### Affected areas

UserListPanel, ContactsColumnView

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

[Kazam_screencast_00010.webm](https://user-images.githubusercontent.com/20650004/192238142-efb1e547-0ea0-49d6-a349-ef184d527686.webm)

### CODE REVIEW

File: ui/app/AppLayouts/Chat/panels/UserListPanel.qml [hunk 1]
```diff
@@ -53,61 +50,74 @@ Item {
...
+        clip: true
+
+        StatusListView {
+            id: userListView
+            objectName: "userListPanel"
+
+            clip: false
+
+            anchors.fill: parent
+            anchors. bottomMargin: Style.current.bigPadding
```
Comment (faust4exe) [inline ui/app/AppLayouts/Chat/panels/UserListPanel.qml:64]: Extra space ? `anchors. bottomMargin`

**Review by faust4exe — APPROVED** [review]
Looks good , just a small comment

File: ui/app/AppLayouts/Chat/panels/UserListPanel.qml [hunk 2]
```diff
@@ -53,61 +50,74 @@ Item {
...
-                const isCurrentUser = model.pubKey === root.rootStore.getPubkey()
-                const visibility = Global.privacyModuleInst.profilePicturesVisibility
-
-                if (isCurrentUser
-                        || visibility === Constants.profilePicturesVisibility.everyone
-                        || (visibility === Constants.profilePicturesVisibility.contactsOnly && model.isContact))
-                    return model.icon
-
-                return ""
+        clip: true
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/panels/UserListPanel.qml:55]: Is this clipping intentional? (with the clip off in the ListView below)?

**Review by caybro — COMMENTED** [review]
Generally looks good, some questions though

File: ui/app/AppLayouts/Chat/panels/UserListPanel.qml (hunk 2 above)
Comment (micieslak) [reply ui/app/AppLayouts/Chat/panels/UserListPanel.qml:55]: yes, it's intentional, otherwise setting `displayMarginEnd` would have no visual effect.

**Review by caybro — APPROVED** [review]

**Review by elina2015 — APPROVED** [review]
Tested and approved

**Pull request merged**